src/System.Reflection.Metadata/src/System.Reflection.Metadata.csproj

```diff
@@ -12,18 +12,8 @@
     <NoWarn>1591</NoWarn>
     <CLSCompliant>false</CLSCompliant>
     <AssemblyVersion>1.3.0.0</AssemblyVersion>
-    <!-- TODO: Remove use of FUTURE -->
-    <DefineConstants>$(DefineConstants);SRM;FUTURE</DefineConstants>
+    <DefineConstants>$(DefineConstants);SRM</DefineConstants>
   </PropertyGroup>
-  <ItemGroup>
-    <PackageDestination Include="lib/netstandard1.1">
-      <TargetFramework>netstandard1.1</TargetFramework>
-    </PackageDestination>
-    <!-- Support targets that were supported in previous package versions -->
-    <PackageDestination Include="lib/portable-net45+win8">
-      <TargetFramework>portable-net45+win8</TargetFramework>
-    </PackageDestination>
-  </ItemGroup>
```

Comment:
Why is this deleted? I don't know how this works (deferring to @ericstj), but doesn't seem related.